lib/cocoapods/project.rb

```diff
@@ -40,6 +40,15 @@ def initialize(path, skip_initialization = false,
     #
     attr_reader :development_pods
 
+    # Overridden to generate UUIDs in a much faster way, since we don't need to check for collisions
+    # (as the Pods project is regenerated each time, and thus all UUIDs will have come from this method)
+    def generate_available_uuid_list(count = 100)
```

Comment:
:grimacing: :grimacing: 

(this is technically fine, but also sad.)

Reply:
SecureRandom is slow!